src/chrome/content/rules/ncar.cc.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	MCB:
+		bbs.ncar.cc
+-->
+
+<ruleset name="ncar.cc">
+
+	<target host="bbs.ncar.cc" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/$" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/.+\.(html|php)" />
```

Comment:
@ivysrono: That particular issue might mean that we will have to let some of these kinds of exclusions throught, but for now, just replace the unescaped dot by a suitable character class, such as `[\w/-]`, or whatever might be relevant for the pattern.